.buildkite/scripts/package.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+VERSION=$(awk '/const DefaultVersion/{print $NF}' version/version.go | tr -d '"')
+PLATFORM_TYPE=$(uname -m)
+MATRIX_TYPE="$1"
+INFRA_REPO="https://github.com/repos/elastic/infra/contents"
+
+if [[ ${BUILDKITE_BRANCH} == "main" && ${MATRIX_TYPE} == "staging" ]]; then
+    echo "INFO: staging artifacts for the main branch are not required."
+else
+    PLATFORMS=""
+    PACKAGES=""
+    if [[ ${PLATFORM_TYPE} == "arm" || ${PLATFORM_TYPE} == "aarch64" ]]; then
+        PLATFORMS="linux/arm64"
+        PACKAGES="docker"
+    fi
+
+    add_bin_path
+    with_go
+    with_mage
+
+    if [[ ${MATRIX_TYPE} == "staging" ]]; then
```

Comment:
Looking at the Jenkinsfile https://github.com/elastic/fleet-server/blob/820ae55a8652775a2ab7a6e98325ca4c4c471ddb/.ci/packaging.groovy#L169C14-L178

It looks like the `make` target depending on the type are `release-manager-release` or `release-manager-snapshot`

Reply:
Yes, you are right, I implemented there the same logic. There is only one difference - `SNAPSHOT=true` in case `matrix_type != "staging"`
https://github.com/elastic/fleet-server/blob/820ae55a8652775a2ab7a6e98325ca4c4c471ddb/Makefile#L260-L266